File: ljh/java/src/level1/test/RemoveSmallestNumberTest.java
package level1.test;

import static org.junit.Assert.assertArrayEquals;

import org.junit.Before;
import org.junit.Test;

import level1.RemoveSmallestNumber;

public class RemoveSmallestNumberTest {
	private int[][] arr;
	private int[][] answers;
	private RemoveSmallestNumber solution;
	
	@Before
	public void setUp() throws Exception {
		arr = new int[][]{{4,3,2,1},{10}};
		answers = new int[][] {{4,3,2}, {-1}};
		solution = new RemoveSmallestNumber();
	}
	
	@Test
	public void test() {
		for(int i=0; i<answers.length; i++) {
			assertArrayEquals(solution.solution(arr[i]), answers[i]);
		}
	}
	
	@Test
	public void test2() {
		for(int i=0; i<answers.length; i++) {
			assertArrayEquals(solution.solution2(arr[i]), answers[i]);
		}
	}
}

File: ljh/javascript/level1/test/RemoveSmallestNumber.test.js
import RemoveSmallestNumber from '../RemoveSmallestNumber';
test(`제일 작은 수 제거하기`, ()=>{
    expect(new RemoveSmallestNumber().solution([4,3,2,1])).toEqual([4,3,2]);
})

test(`제일 작은 수 제거하기`, ()=>{
    expect(new RemoveSmallestNumber().solution([10])).toEqual([-1]);
})